lib/routes/account.js

```diff
@@ -922,7 +966,9 @@ module.exports = function (
             // There's code in the handler that checks for a valid email,
             // no point adding overhead by doing it again here.
             email: isA.string().required(),
-            verified: isA.boolean().required()
+            verified: isA.boolean().required(),
+            sessionVerified: isA.boolean().required(),
```

Comment:
FWIW, I hope that we can engineer our way out of overloading the `/recovery_email/status` endpoint like this, well before we add any other challenge types.  Having this endpoint do double duty checking both account and token verification is a backwards-compatibility hack.